Tests/DefaultXRPClientTest.swift

```diff
@@ -458,7 +458,7 @@ final class DefaultXRPClientTest: XCTestCase {
       .testTransactionHistoryResponse
       .transactions
       .map { transactionResponse in
-        return XRPTransaction(transaction: transactionResponse.transaction)
+        return XRPTransaction(getTransactionResponse: transactionResponse)
```

Comment:
Why did this change from `transaction` to `getTransactionResponse`?

Reply:
a `GetTransactionResponse` contains a `Transaction`.  Originally, a `Transaction` had everything we needed to represent a transaction on the XRPL.  However, we realize that we actually want our notion of a transaction to include some metadata (such as the fields being added in this PR) which are only available in the surrounding object, a `GetTransactionResponse`.  Thus the change.